MAUI/ImageEditor/text.md

```diff
@@ -88,6 +89,38 @@ Customize the appearance of text annotation using the [`ImageEditorTextSettings`
 
 ![Text annotation customization in .NET Maui ImageEditor](images/text/imageeditor-text-customization.png)
 
+## Customize the background color of text annotations
+
+The background color of the text annotations can be customized by using the `Background` property.
```

Comment:
The background color of the text annotations can be customized using the `Background` property.

Reply:
Updated